hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/SelectiveWorkBalancerTest.java

```diff
@@ -132,8 +160,13 @@ public void shouldRebalanceAfterConsumerDisappearing() {
         // given
         List<String> supervisors = ImmutableList.of("c1", "c2");
         List<SubscriptionName> subscriptions = someSubscriptions(2);
-        WorkloadConstraints constraints = WorkloadConstraints
-                .defaultConstraints(CONSUMERS_PER_SUBSCRIPTION, MAX_SUBSCRIPTIONS_PER_CONSUMER, supervisors.size());
+        WorkloadConstraints constraints = new WorkloadConstraints(
```

Comment:
Minor: What do You think about extracting method to create new WorkloadConstraints? Same code occurs multiple times

Reply:
I would like to keep the test setup as close as possible to assertions and the logic being tested - to show the reader how given parameters affect the final result. Therefore, I wanted to avoid something like: 
```java
WorkloadConstraints constraints = createWorkloadConstraints(supervisors.size());
```
with values of the parameters `consumersPerSubscription` and `maxSubscriptionsPerConsumer` being hidden. 

Instead, I introduced a builder that allows us to skip setting parameters that are irrelevant for a given test case.

Please let me know what you think about it.